packages/foundry/contracts/witch/WitchV2.sol

```diff
@@ -110,17 +121,44 @@ contract WitchV2 is AccessControl {
         emit LimitSet(ilkId, baseId, max);
     }
 
+    /// @dev Governance function to set other liquidation contracts that may have taken vaults already.
+    /// @param a The address that may be set/unset as another witch
+    /// @param isWitch Is this address a witch or not
+    function setAnotherWitch(address a, bool isWitch) external auth {
```

Comment:
```suggestion
    function setAnotherWitch(address witch, bool isWitch) external auth {
```

Reply:
Didn't wanna call it witch cause if isWitch is false makes no sense, I called it a for address cause we don't really know if it's a witch or not, but happy to change it if you feel strong about it

Reply:
Between `a` and `witch` I prefer the latter. You can also use something more generic such as `target` or `value` as long as it's a word.